ci/docs/build.sh

```diff
@@ -41,7 +41,9 @@ conda list --show-channel-urls
 
 # Build Doxygen docs
 gpuci_logger "Build Doxygen docs"
-"$PROJECT_WORKSPACE/build.sh" cppdocs -v
+pushd cpp
+doxygen Doxyfile.in
+popd
```

Comment:
```suggestion
cd "$PROJECT_WORKSPACE/cpp"
doxygen Doxyfile.in
```

This particular script runs in our `devel` container, which starts in the `/rapids/notebooks` directory, so this `pushd` command won't place you in the correct directory.

For this reason, let's stick with the `cd` convention that's being used in this script.